Tests/RedisTests/RedisTests.swift

```diff
@@ -182,5 +236,15 @@ class RedisTests: XCTestCase {
         ("testStringCommands", testStringCommands),
         ("testListCommands", testListCommands),
         ("testExpire", testExpire),
+        ("testSetCommands", testSetCommands),
     ]
 }
+
+#if swift(>=4.2)
```

Comment:
https://github.com/vapor/redis/blob/d9dbce13bc967c2b3a155a17dea2e9d13bbe21b3/Tests/RedisTests/RedisTests.swift#L243-L244

These two lines could be written as `#if !swift(>=4.2)`

Reply:
D'oh! Made this section simpler